branding/src/test/scala/com/normation/plugins/branding/api/BrandingApiTest.scala

```diff
@@ -0,0 +1,87 @@
+package com.normation.plugins.branding
+
+import better.files._
+import com.normation.plugins.branding.api.BrandingApi
+import com.normation.plugins.branding.api.BrandingApiService
+import com.normation.rudder.api.ApiVersion
+import com.normation.rudder.rest.RestTestSetUp
+import com.normation.rudder.rest.TraitTestApiFromYamlFiles
+import java.nio.file.Files
+import net.liftweb.common.Loggable
+import org.junit.runner.RunWith
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+import org.specs2.specification.AfterAll
+
+@RunWith(classOf[JUnitRunner])
+class BrandingApiTest extends Specification with TraitTestApiFromYamlFiles with Loggable with AfterAll {
+  sequential
+
+  val restTestSetUp = RestTestSetUp.newEnv
+
+  val tmpDir: File = File(Files.createTempDirectory("rudder-test-"))
```

Comment:
what we often do is create a directory, and clean it `AfterAll` but only if a jvm parameter is not set, something like: 
```
  override def afterAll(): Unit = {
    if (System.getProperty("tests.clean.tmp") != "false") {
      // do cleanup
    }
  }
```

That allow to keep the information when debugging is needed, you run the test with `-Dtests.clean.tmp=false` and you can explore what is going wrong. 

It's not a mandatory change, and since I think nothing is written by the business code tested here, it's not really interesting (it's just fyi)